src/GraphQL.NewtonsoftJson/ExecutionResultJsonConverter.cs

```diff
@@ -121,50 +119,7 @@ private void WriteErrors(JsonWriter writer, ExecutionErrors? errors, JsonSeriali
 
             writer.WritePropertyName("errors");
 
-            writer.WriteStartArray();
-
-            foreach (var error in errors)
-            {
-                var info = _errorInfoProvider.GetInfo(error);
```

Comment:
All the code here (within the loop) was directly moved to the new class.  Ditto with SystemTextJson.